plugin/pkg/scheduler/scheduler.go

```diff
@@ -111,12 +111,10 @@ func (s *Scheduler) scheduleOne() {
 
 	// Optimistically assume that the binding will succeed and send it to apiserver
 	// in the background.
-	// The only risk in this approach is that if the binding fails because of some
-	// reason, scheduler will be assuming that it succeeded while scheduling next
-	// pods, until the assumption in the internal cache expire (expiration is
-	// defined as "didn't read the binding via watch within a given timeout",
-	// timeout is currently set to 30s). However, after this timeout, the situation
-	// will self-repair.
+	// If the binding fails because of some reasons, scheduler will release the
```

Comment:
"If the binding fails, scheduler will release resources allocated to assumed pod immediately."